sdk/python/feast/feature_store.py

```diff
@@ -758,9 +758,16 @@ def materialize_incremental(
             feature_views_to_materialize = self._list_feature_views(
                 hide_dummy_entity=False
             )
+            feature_views_to_materialize = [
+                fv for fv in feature_views_to_materialize if fv.online is True
```

Comment:
Hi @judahrand, I think `is True` and `is False` could be dropped here, since `online` property is always `boolean` you can simply use `if fv.online:` or `if not fv.online`

Reply:
Made that change.